cfgov/unprocessed/apps/erap/css/main.less

```diff
@@ -0,0 +1,12 @@
+#rental-assistance-finder {
```

Comment:
It would be nice to pull this from the CSS distributed in the rental-assistance-finder npm module, but we probably want to update the code in the RAFT repo to generate a single `main.css` first.